src/amo/components/SearchSortLink/index.js

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+import { makeQueryString } from 'core/api';
+import { convertFiltersToQueryParams } from 'core/searchUtils';
+import Link from 'amo/components/Link';
+
+import './SearchSortLink.scss';
+
+
+export function queryString(filters) {
+  return makeQueryString(convertFiltersToQueryParams(filters));
+}
+
+export default class SearchSortLink extends React.Component {
```

Comment:
I don't feel like these small components really need to be in their own file. It does make the main component a bit smaller though.

Perhaps it would be better to include this within `components/SearchSort` instead of giving it its own component directory? That way it's sort of an internal component.

```
amo/components/SearchSort/index.js
amo/components/SearchSort/SearchSort.scss
amo/components/SearchSort/SearchSortLink.js
amo/components/SearchSort/SearchSortLink.scss